alibi_detect/utils/registry.py

```diff
@@ -0,0 +1,24 @@
+import catalogue
+from alibi_detect.utils.frameworks import has_pytorch, has_tensorflow
+
+if has_tensorflow:
+    from alibi_detect.utils.tensorflow.kernels import GaussianRBF as GaussianRBF_tf
+    from alibi_detect.cd.tensorflow import preprocess_drift as preprocess_drift_tf
+    from alibi_detect.utils.tensorflow.data import TFDataset as TFDataset_tf
+
+if has_pytorch:
+    from alibi_detect.utils.pytorch.kernels import GaussianRBF as GaussianRBF_torch
+    from alibi_detect.cd.pytorch import preprocess_drift as preprocess_drift_torch
+
+# Create registry
+registry = catalogue.create("alibi_detect", "registry")
+
+# Register alibi-detect classes/functions
+if has_tensorflow:
+    registry.register('utils.tensorflow.kernels.GaussianRBF', func=GaussianRBF_tf)
```

Comment:
Have kept population of the registry very simple for now. We'd likely want to readdress this if/when we wish to pre-register more functions/classes.

 On another related topic, I quite like the idea of adding thin wrappers for `GaussianRBF` and `preprocess_Drift`, which would select the correct backend-specific one. This means we could remove the burden on the user to pick the correct backend-specific option.